cli/src/program.rs

```diff
@@ -1,3 +1,5 @@
+use solana_cli_output::{CliAccount, CliAccountNewConfig};
```

Comment:
nit: I think this will get tripped during CI -- can you fold this into the larger `use` statement below?